packages/tpetra/core/test/Behavior/Behavior_Default.cpp

```diff
@@ -61,25 +61,9 @@ namespace {
 
 TEUCHOS_UNIT_TEST(Behavior, Default)
 {
-#ifdef HAVE_TPETRA_DEBUG
-  bool debug_default = true;
-#else
-  bool debug_default = false;
-#endif
-  bool dbg = Tpetra::Details::Behavior::debug();
-  TEUCHOS_TEST_ASSERT(dbg==debug_default, out, success);
-
   bool verbose_default = false;
   bool verb = Tpetra::Details::Behavior::verbose();
   TEUCHOS_TEST_ASSERT(verb==verbose_default, out, success);
-
-#ifdef TPETRA_ASSUME_CUDA_AWARE_MPI
-  bool cuda_aware_mpi_default = true;
-#else
-  bool cuda_aware_mpi_default = false;
-#endif
-  bool cuda_aware_mpi = Tpetra::Details::Behavior::assumeMpiIsCudaAware();
```

Comment:
Will we add back the test for this function at some point?  If we end up getting rid of the CMake option, then we still need some default value.